code/noise.py

```diff
@@ -0,0 +1,36 @@
+import time
+
+from talon import ctrl
+from talon import tap
+from talon.audio import noise
```

Comment:
Would you mind updating it to use the new-api style of noise, similar to the below snippet? 

`from talon import noise
def on_pop(active):
    ctrl.mouse_click(button=0, hold=16000)
noise.register('pop', on_pop)`